docs/source/user_guide/automl.ipynb

```diff
@@ -642,6 +642,152 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### [IterativeAlgorithm](https://evalml.alteryx.com/en/stable/autoapi/evalml/automl/automl_algorithm/iterative_algorithm/index.html)\n",
+    "\n",
+    "`IterativeAlgorithm` is the first AutoML Algorithm created in EvalML and can be acessed with the `search_iterative` method or specifiying `AutoMLSearch(automl_algorithm='iterative')`. The algorithm works as follows:\n",
+    "\n",
+    "- Every batch (after the initial baseline model) contains pipelines of all available estimators for the specified problem type\n",
+    "- Pipelines contain preprocessing (imputing, encoding, etc.) needed for machine learning but no feature selection is applied\n",
+    "- ensembling can be turned on by passing in the `ensembling=True` parameter and will be run after a whole cycle of training has occurred (each allowed pipeline trains for one batch)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import evalml\n",
+    "X, y = evalml.demos.load_fraud(n_rows=250)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.automl import search_iterative\n",
+    "\n",
+    "# top level search method will run `AutoMLSearch` with `IterativeAlgorithm` as well as apply our default data checks\n",
+    "auto_iterative, messages_iterative = search_iterative(X, y, problem_type='binary')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml import AutoMLSearch\n",
+    "auto_iterative = AutoMLSearch(X_train=X, y_train=y, problem_type='binary', automl_algorithm='iterative', verbose=True)\n",
+    "auto_iterative.search()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### [DefaultAlgorithm](https://evalml.alteryx.com/en/stable/autoapi/evalml/automl/automl_algorithm/default_algorithm/index.html)\n",
+    "\n",
+    "`DefaultAlgorithm` was designed to do three main things:\n",
+    "\n",
+    "1. Abstract out more parameters and decisions from the user.\n",
+    "\n",
+    "2. Perform deeper tuning for high performing pipelines.\n",
+    "\n",
+    "3. Create a platform to introduce feature selection as well as other potential techniques/heuristics for AutoML.\n",
+    "\n",
+    "`DefaultAlgorithm` does this by creating the concept of two modes: `fast` and `long` where `fast` is a subset of long. The algorithm runs as follows:\n",
+    "\n",
+    "1. Run naive pipelines:\n",
+    "    a. run naive linear model with default preprocessing pipeline\n",
+    "    b. run basic random forest pipeline with default preprocessing pipeline\n",
+    "    \n",
+    "\n",
+    "2. Run naive pipelines with feature selection. Subsequent pipelines will use the selected features with a SelectedColumns transformer.\n",
```

Comment:
This part is still a little unclear to me. Are the naive pipelines the linear model and the random forest model? If so, it might be clearer to do:
```
1. Run naive pipelines:
   a. a linear model with the default preprocessing pipeline
   b. a random forest pipeline with the default preprocessing pipeline
2. Run the same pipelines, this time with feature selection. Subsequent pipelines...
```